examples/demo/main.go

```diff
@@ -41,12 +42,8 @@ var (
 )
 
 func main() {
-	cfg, err := config.Read()
-	if err != nil {
-		logger.WithFields(logrus.Fields{
-			"error": err.Error(),
-		}).Fatalf("cannot read configuration.")
-	}
+	cfg := config.Read()
```

Comment:
This causes a significant reliability issue where pushing a bad config will take down your entire cluster. On startup it makes sense to fail because you cannot read the config but existing servers should report a read failure and continue running with the old configuration.